application.js

```diff
@@ -8,46 +8,66 @@
 function runChallenges() {
 
   // Ex 1. Read the content of the email input
-  var email = null;  // TODO: replace null, keep the email variable.
+  var email = document.getElementById('email').value;  // TODO: replace null, keep the email variable.
 
 
   // Ex 2. Fill the content of the email input with your email
+  document.getElementById('email').value = "k.perachon@gmail.com";
 
 
 
   // Ex 3. Replace the email hint (next to the input) with 'This is my email now'
   //       The text should be emphasized using a <strong> tag
-
+  document.getElementById('email-hint').innerHTML = "<strong>This is my email now</strong>";
 
   // Ex 4. Add the .blue CSS class to the table header cells (th elements)
+  $('th').addClass('blue');
 
 
 
   // Ex 5. Count the number of table body rows there are (team count!)
-  var team_count = 0;  // TODO: replace 0, keep the team_count variable.
-
+  var team_count = $('tbody > tr').length;  // TODO: replace 0, keep the team_count variable.
 
 
   // Ex 6. Say there is a 15th team added to the table.
   //       Add a row at the bottom, this new team should have zero points.
+  var table = $('table')[0];
+  var newRow   = table.insertRow(15);
+  var newCell  = newRow.insertCell(0);
+  var newText  = document.createTextNode('15');
+  newCell.appendChild(newText);
 
+  newCell  = newRow.insertCell(1);
+  newText  = document.createTextNode('Nouvelle équipe de dingue');
+  newCell.appendChild(newText);
+
+  newCell  = newRow.insertCell(2);
+  newText  = document.createTextNode('0');
+  newCell.appendChild(newText);
```

Comment:
There is a simplier solution

    var row = $('<tr>').html('<td>' + (teamCount + 1) + '</td><td>New team</td><td>0</td></tr>');
    $('tbody').append(row);